frontend/include/chpl/parsing/parser-error-classes-list.h

```diff
@@ -50,6 +50,7 @@ ERROR_CLASS(DisallowedControlFlow,
 POSTPARSE_ERROR_CLASS(MultipleManagementStrategies, const uast::New::Management,
                       const uast::New::Management)
 ERROR_CLASS(IllegalUseImport, const uast::AstNode*, const uast::AstNode*)
+ERROR_CLASS(UnsupportedAsIdent, const uast::As*, const uast::AstNode*)
```

Comment:
Just realized that these break the alphabetic ordering.

Reply:
On that note, it looks like when we split the massive original `error-classes-list.h` we left the note about alphabetic ordering in the original file and did not replicate it into the separate `parser` and `resolution` versions -- that seems like a reasonable thing to fix in this PR.

Reply:
@mppf - could you please also check that the source files are sorted?